modules/nf-core/vt/decomposeblocksub/tests/main.nf.test

```diff
@@ -0,0 +1,62 @@
+
+nextflow_process {
+
+    name "Test Process VT_DECOMPOSEBLOCKSUB"
+    script "../main.nf"
+    process "VT_DECOMPOSEBLOCKSUB"
+    config "./nextflow.config"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "vt"
+    tag "vt/decomposeblocksub"
+
+    test("test-vt-decomposeblocksub") {
+
+        when {
+            process {
+                """
+                input[0] = [
+				[ id:'test', single_end:false ], // meta map
```

Comment:
Could you add an indentation here ? and for the other lines inside `input[0]` in the 2 tests ?
```suggestion
				    [ id:'test', single_end:false ], // meta map
```